tgno/settings/base.py

```diff
@@ -176,6 +184,26 @@
 
 # Base URL to use when referring to full URLs within the Wagtail admin backend -
 # e.g. in notification emails. Don't include '/admin' or a trailing slash
-WAGTAILADMIN_BASE_URL = env("WAGTAILADMIN_BASE_URL", "http://example.com")
+WAGTAILADMIN_BASE_URL = env("WAGTAILADMIN_BASE_URL", "http://localhost:8000")
+WAGTAILAPI_BASE_URL = env("WAGTAILAPI_BASE_URL", "http://localhost:8000")
+
+# Custom settings
+SITE_NAME = env("SITE_NAME", "TG.no")
+DISABLE_LOCAL_AUTH = env("DISABLE_LOCAL_AUTH", "False").lower() in ("true", "1")
+
+# Django Social Auth
+AUTHENTICATION_BACKENDS = []
+
+SOCIAL_AUTH_JSONFIELD_ENABLED = True
+SOCIAL_AUTH_KEYCLOAK_KEY = os.getenv("SOCIAL_AUTH_KEYCLOAK_KEY") or None
```

Comment:
is there a reason for this to be different from the once belove?

Reply:
Well, yes and no. I need it to be `None` when the variable is set but empty.
But there is no reason we can't do the same for all others. I just think the `env()` is more elegant.

I guess I could have moved the "if empty or not set" check where we use it.

Reply:
So this `SOCIAL_AUTH_KEYCLOAK_SECRET = env("SOCIAL_AUTH_KEYCLOAK_SECRET", None)` will sett it to `""` if it's empty and `SOCIAL_AUTH_KEYCLOAK_KEY = os.getenv("SOCIAL_AUTH_KEYCLOAK_KEY") or None` will sett it to `None` when it's empty?

Reply:
>  I just think the `env()` is more elegant.

I also think `env()` is more elegant 😅 
That's why I'm asking 😄

Reply:
Yes. I use SOCIAL_AUTH_KEYCLOAK_KEY if we should enable keycloak login or not. Do not want to enable it if the user is just copy/pasting the example .env file.